CONTRIBUTING.md

```diff
@@ -47,47 +47,48 @@ First, ensure that [Yarn](https://yarnpkg.com/) and
 
 ## The demo project
 
-The [demo project](demo) was initially forked from
-[dvc-checkpoints-mnist](https://github.com/iterative/dvc-checkpoints-mnist/tree/make_checkpoint),
-and is provided as a lightweight, convenient testbed to try the extension out
-with. It is not an exhaustive showcase of DVC's features, so testers are
-encouraged to try other DVC repositories -- especially real-world cases!
+The [demo project](demo) is provided as a lightweight, convenient testbed to try
+the extension out with. It is not an exhaustive showcase of DVC's features, so
+testers are encouraged to try other DVC repositories -- especially real-world
+cases!
 
-- Go to the `demo/` directory of this repo.
+- Install [Yarn](https://yarnpkg.com/) if needed. Run `yarn setup:venv` from the
+  root of this repo to set up a virtual Python environment for the demo project
+  (in `demo/.env`).
 
-- It is recommended that you create the project's virtual environment with
-  Python 3.8 as some dependencies are not yet available for Python 3.9
+- Important: Open the `demo/` folder of this repo directly in VS Code.
 
-- `yarn install` is a prerequisite for automatically setting up the demo
-  project.
+- Pull the project data with the extension (either from the **DVC Tracked**
+  panel in the _File Explorer_ or the **DVC panel** in the _Source Code_ view)
+  or by running `dvc pull` from a VS Code _Terminal_.
 
-- Run `yarn setup:venv` from the monorepo root (not `./demo`) to automatically
-  setup the Python virtual environment in the demo project.
+  > Remember to activate the virtual env first (`source .env/bin/activate`) if
+  > you're using your system's terminal.
 
-- Once the environment has been setup you should navigate into the demo folder
-  activate the virtual environment and run `dvc pull`.
+- In order to run experiments in the demo project (from the **DVC View**), the
```

Comment:
That's what this is called, I think?

<img width="218" alt="image" src="https://user-images.githubusercontent.com/1477535/175184107-9e4e89d7-f214-42a1-bb11-d244b55babaa.png">

Reply:
per https://code.visualstudio.com/docs/getstarted/userinterface#_views

Reply:
Called `dvc-views` under `views` in package.json which is slightly confusing (technically it's a single view with multiple panels for the side bar)

Reply:
We call it `dvc.viewContainer` (under `contributes.walkthroughs`) and file .../walkthrough/view-container.md but I think "container" is a term we came up with. I'm not seeing that in VS Code docs.

This is all irrelevant to the user of course but it may be good to decide on our own terminology. I vote for just "view" since it matches VS Code docs; I'm happy to update all these internals to singular `dvc(-view)` along with a walkthrough copy edit if needed.

Reply:
How would you call then actual Views (Experiments, Sort By, Filter, etc) ?

Again, it's probably a bigger discussion. not relevant to this PR ... my comment was not about this at all

Reply:
Those are _Panels_ in the _Activity Bar_ of the _DVC View_.

Going back in the text in question, I guess you can run experiments both from several places (Experiments panel or table, or command palette) so no need to mention DVC View specifically... Linking from "run experiments" now instead. PTAL

Reply:
I don't think they are panels. We can check the API or sources to see who it's called.

Reply:
https://code.visualstudio.com/api/ux-guidelines/views#view-containers

Reply:
I see, thanks for clarifying VS Code UX terminology. And I see that `dvc-views` is also under `contributes .viewsContainers` indeed... So it's a _View Container_ for the _Primary Sidebar_.

Anyway, for now I just removed the reference and linked to the Run Experiments walkthroughs (PTAL).

But should we use official VS Code terminology (from the dev-facing docs) for README and walkthrough text going forward? I'd need to update "DVC View" in a few places now. Or do we want to have our own terminology like "click the **DVC** button in the _Activity Bar_" ?

Reply:
> I guess there's some discrepancies

<img width="344" alt="Screen Shot 2022-06-24 at 5 55 26 PM" src="https://user-images.githubusercontent.com/3659196/175752007-0a379295-963d-475c-ace1-c74620bff14d.png">

no, there is no discrepancy I think. Most of the things (all?) in this list are Views. It just happened that in certain cases they are the only Views by default in the containers.